### PR TITLE
{lib}[foss/2017a] zlib v1.2.11

### PR DESCRIPTION
**Pull request by reedts**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/z/zlib/zlib-1.2.11-foss-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'http://www.zlib.net/'
+description = """zlib is designed to be a free, general-purpose, legally unencumbered -- that is,
+ not covered by any patents -- lossless data-compression library for use on virtually any
+ computer hardware and operating system."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/libpng/files/zlib/%(version)s', 'download')]
```
Comment (boegel) [inline easybuild/easyconfigs/z/zlib/zlib-1.2.11-foss-2017a.eb]: @reedts Any reason you changed this, the other zlib 1.2.11 easyconfigs are using `http://zlib.net/fossils` as source URL?

**Comment by boegel** [comment]
@reedts Is there a particular reason you are adding this, rather than using `zlib-1.2.11-GCCcore-6.3.0.eb` which will be picked up to provide a zlib 1.2.11 installation that is compatible with `intel/2017a`?

**Comment by reedts** [comment]
Given that 1.2.11 exists and can easily be built for foss and intel toolchains I do not need it explicitly for foss-2017a. We were just upgrading our foss toolchain and I could not find an appropriate version then.

**Comment by boegel** [comment]
@reedts So, shall we close this PR then?

**Comment by reedts** [comment]
@boegel yes, you can close the PR.

**Pull request closed without merging**